buildman/manager/executor.py

```diff
@@ -390,17 +423,30 @@ def __init__(self, *args, **kwargs):
             "BUILDER_VM_CONTAINER_IMAGE", "quay.io/quay/quay-builder-qemu-coreos:stable"
         )
 
+    def validate(self, executor_config):
+        try:
+            create_job = self._build_request(executor_config, "POST", self._jobs_path(), json={})
+            if int(create_job.status_code / 100) != 400:
```

Comment:
Should this be `int(status_code / 100) == 4`?

Reply:
No, because then it would succeed on auth errors, which we're explicitly trying to catch.

Reply:
Wouldn't the current check return a single digit though? 
e.g `401 // 100` is `4`, not `400`

Reply:
Oh... good catch. Copy-paste issue.